component/main.jsonnet

```diff
@@ -66,12 +68,34 @@ local readServices = kube.ClusterRole('appcat:services:read') + {
   ],
 };
 
-{
-  '10_clusterrole_view': xrdBrowseRole,
-  [if isOpenshift then '10_clusterrole_finalizer']: finalizerRole,
-  '10_clusterrole_services_read': readServices,
+// adding namespace for syn-appcat
+local ns = kube.Namespace('syn-appcat');
+// adding missing namespace for slos
```

Comment:
The SLI part of the component deploys this one already. Why did you add it again here?

Reply:
because when You setup fresh kind cluster, then namespace is never created and our makefile returns error

Reply:
What target fails for you? The `vshnpostgresql` target works for me without error.

But I would still remove it here, or we will have problems with argoCD once we deploy stuff. There should be other means to solve your issue.